src/NServiceBus.Core/Unicast/RunningEndpointInstance.cs

```diff
@@ -61,32 +63,64 @@ public async Task Stop()
 
         public Task Send(object message, SendOptions options)
         {
+            Guard.AgainstNull(nameof(message), message);
+            Guard.AgainstNull(nameof(options), options);
+
+            CheckIfInRunnungsState();
```

Comment:
Typo "Runnungs"

Perhaps a better name would be GuardAgainstUseWhenStopped ?

Reply:
Renamed to GuardAgainstUseWhenNotStarted